src/System.Net.Http/src/System/Net/Http/HttpClientHandler.cs

```diff
@@ -24,15 +24,16 @@ private static bool UseSocketsHttpHandler
                     return useSocketsHttpHandler;
                 }
 
-                // AppContext switch wasn't used. Check the environment variable to see if it's been set to true.
+                // AppContext switch wasn't used. Check the environment variable to determine which handler should be used.
```

Comment:
It seems like the standard way of doing app context switches is to create our own `LocalAppContextSwitches` per assembly and set things up there. See the [System.Drawing](https://github.com/dotnet/corefx/blob/master/src/System.Drawing.Common/src/System/Drawing/LocalAppContextSwitches.cs) and [System.Xml](https://github.com/dotnet/corefx/blob/master/src/System.Private.Xml/src/System/Xml/Core/LocalAppContextSwitches.cs) implementations for reference. I know that this isn't code that you added, but I think that we should probably go ahead and move the app context switch above this so as to match the standard format.

Reply:
Why is that valuable?
cc: @AlexGhiondea

Reply:
I actually don't know why that's the standard practice in Framework. The only reason I can think of is to keep all of the app context switches in one location. If that isn't compelling enough though it's something I'm fine with dropping.

Reply:
Ok. Please feel free to open an issue for it, but let's not block this PR on it.